wellregistry/registry/admin/monitoring_location.py

```diff
@@ -546,5 +546,5 @@ def has_delete_permission(self, request, obj=None):
 
     def changelist_view(self, request, extra_context=None):
         self.message_user(request,
-                          'Note: additions/changes to monitoring locations may take up to a day to appear in NGWMN')
+                          'Note: additions/changes to monitoring locaitons may take up to a day to appear in the NGWMN Data Portal')
```

Comment:
Typo in locations needs to be fixed (looks like the typo was in the ticket).

Reply:
fixed